examples/json-mapping-example/README.md

```diff
@@ -87,3 +89,19 @@ occur multiple times under "*:/person/*:names".
 1. As shown in the "prefix" mapping, XML attributes can be referenced via the "@" symbol. 
 
 Otherwise, all of the features used in the jsonToJsonFlow can be used in this mapping as well. 
+
+### Mapping features in the jsonToXml flow
+
+From the "Flows" view in QuickStart, open the jsonToXml flow and select the mapping step. QuickStart will display the
+JSON document in staging in the "Source Data" panel and the mapping in the "Entity" panel. The mapping displays all of
+the properties of the Person entity along with its nested Name and Address entities. Each row in the mapping is similar to the jsonToJson flows
```

Comment:
This sentence should end with "jsonToJson flow." as opposed to "flows" with no period.